data/json/recipes/recipe_vehicle.json

```diff
@@ -537,6 +537,20 @@
     "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "SAW_M", "level": 1 } ],
     "components": [ [ [ "pipe", 2 ] ], [ [ "wheel_bicycle", 2 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "wheel_rail_small_pair",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_PARTS",
+    "skill_used": "fabrication",
+    "skills_required": [ "mechanics", 1 ],
+    "difficulty": 3,
+    "time": "50 m",
+    "autolearn": true,
+    "using": [ [ "blacksmithing_standard", 5 ], [ "steel_standard", 12 ] ],
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "SAW_M", "level": 1 } ],
```

Comment:
HAMMER quality is unnecessary here; already included into "blacksmithing_standard" requirement.

Reply:
Thanks, I removed it.